README.md

```diff
@@ -347,6 +348,59 @@ TotalCurrentRevenue.calculate
 AverageDailyRevenue.result
 ```
 
+## Rule and Ruleset
+
+### When to use it
+
+Rule objects provide a place for conditional logic
```

Comment:
We need much better description for that. @bartoszmaka - we can meet to discuss it, if you want